assets/section-image-banner.css

```diff
@@ -318,21 +318,12 @@
 @media screen and (min-width: 750px) {
   .banner--desktop-transparent .banner__box {
     background: transparent;
-    --color-foreground: 255, 255, 255;
-    --color-button: 255, 255, 255;
-    --color-button-text: 0, 0, 0;
```

Comment:
Those changes are on declarations within a media query `@media screen and (min-width: 750px) {` so they're only impacting the wider screens. 
If you're on mobile the colour scheme is never applied and the default is white. 

So it can make for a confusing experience. As it's working on desktop and applying the colour scheme accordingly when a container is and isn't showing but not on mobile. 

Also the info next to the colour scheme setting needs to be updated: 
![](https://screenshot.click/04-25-3ucsx-ooc3e.png)